.ci/checker-framework.groovy

```diff
@@ -113,12 +113,20 @@ private static int checkCheckerFrameworkReport(final String profile) {
 private static List<List<String>> getCheckerFrameworkErrors(final String profile) {
     final List<String> checkerFrameworkLines = new ArrayList<>()
     final String command = "mvn -e --no-transfer-progress clean compile -P${profile}"
-    final Process process = getOsSpecificCmd(command).execute()
-    process.in.eachLine { final line ->
-        checkerFrameworkLines.add(line)
-        println(line)
+    final ProcessBuilder processBuilder = new ProcessBuilder(getOsSpecificCmd(command).split(' '))
+    processBuilder.redirectErrorStream(true)
+    final Process process = processBuilder.start()
+
+    final BufferedReader reader = new BufferedReader(new InputStreamReader(process.inputStream))
+    String lineFromReader = reader.readLine()
+    while (lineFromReader != null) {
+        println(lineFromReader)
+        checkerFrameworkLines.add(lineFromReader)
```

Comment:
Maybe not related to this PR, but why do we need to add everything to an array and not process it in real time? It doesn't seem like we re-use old indexes from the array.

Reply:
Yes, it can be done with an additional boolean control variable in the same loop, let's keep it for a separate PR.

Reply:
Then please work on this. I am marking this resolved for now.